src/webextension/ext-dom.c

```diff
@@ -170,10 +170,23 @@ char *ext_dom_editable_get_value(WebKitDOMElement *element)
     return value;
 }
 
+void ext_dom_focus_element_by_id(WebKitDOMDocument *parent, char *element_id)
+{
+    WebKitDOMElement *elem;
+    GError *error = NULL;
+
+    elem = webkit_dom_document_get_element_by_id(parent, element_id);
+    if (elem != NULL) {
+        webkit_dom_element_set_attribute(elem, "disabled", "false", &error);
```

Comment:
In case of an error the variable `error` have to be freed by `g_error_free()`, but I assume you can give NULL instead of unused var.
But this function does not work for me. I think it would be better to use `webkit_dom_element_remove_attribute` instead, which works for me.